plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConnectorModule.java

```diff
@@ -81,6 +84,13 @@ public static HeaderProvider createHeaderProvider(NodeManager nodeManager)
             return FixedHeaderProvider.create("user-agent", "Trino/" + nodeManager.getCurrentNode().getVersion());
         }
 
+        @Provides
+        @Singleton
+        public static FormatInterpolator<ConnectorSession> labelInterpolator(BigQueryConfig config)
```

Comment:
I don't understand that ask. FormatInterpolator is shared. I'd need to inject BigQueryConfig there (and I don't want to do that). I can bind to instance but then I'd need to a) build config object b) use TypeInfo since this class is generic. This approach is much simpler and more readable.